ui/guest/src/iceRegistry.ts

```diff
@@ -202,7 +205,7 @@ export function getRecordDropTargets(id: number): ICERecord[] {
       // move an item deeper inside itself).
       return rec.modelId === id || allChildren.includes(rec.modelId);
     });
-  } else if (field.type === 'repeat') {
+  } else if (field.type === 'repeat' || (field.type === 'node-selector' && !isModelId)) {
     return getRepeatGroupItemDropTargets(record);
```

Comment:
This seems confusing. How can you look the repeat group drop target for a node selector item? Doesn't make sense to me.

Reply:
As explained in the comment above, when having a node-selector of files, the model lookup contains the actual model of the files (not the ids, as with components), with a `key/value` just as in repeating groups. That's the reason why in that case it gets treated just as a repeating group.